app/controllers/distributions_controller.rb

```diff
@@ -1,5 +1,12 @@
 class DistributionsController < ApplicationController
   skip_before_action :set_distributions
+  before_action :set_parameters
```

Comment:
you are setting parameters for all the actions when you only need them in two actions... 🙈

Reply:
you can do this by:

```ruby
before_action :set_parameters, only: [:index :testing]
```

😉